packages/api-client/src/user/UserAPI.ts

```diff
@@ -52,7 +52,7 @@ type PrekeysResponse = {
   failed_to_list?: QualifiedId[];
 };
 function isPrekeysResponse(object: any): object is PrekeysResponse {
-  return object.qualified_user_client_prekeys && object.failed_to_list;
```

Comment:
Now that `failed_to_list` is optional on backend side, we need to also consider a response without `failed_to_list` as a valid response